LazyEvaluation/LazyEvaluation/LazyInterface.cs

```diff
@@ -0,0 +1,13 @@
+// <copyright file="LazyInterface.cs" company="NematMusaev">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+namespace LazyEvaluation;
+
+/// <summary>
+/// Interface of Lazy evulation classes.
+/// </summary>
+/// <typeparam name="T">General Type.</typeparam>
+public interface ILazy<T>
```

Comment:
```suggestion
public interface ILazy<out T>
```
Если есть возможность указать вариантность, лучше указать.

Reply:
Напоминаю. Не поправлено.